.devcontainer/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
       # This is where VS Code should expect to find your project's source code
       # and the value of "workspaceFolder" in .devcontainer/devcontainer.json
       - ..:/var/www/html
+      - hashtopolis-server-dev:/usr/local/share/hashtopolis:Z
```

Comment:
The ':Z' only seems relevant on systems running selinux policies. (See: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label). Looks optional (jet harmless) here.